internal/database/migrations/migrations.go

```diff
@@ -38,6 +38,7 @@ func Seed(db *gorm.DB) error {
 				Summary:     faker.Hacker().SaySomethingSmart(),
 				Latitude:    float64(faker.Address().Latitude()),
 				Longitude:   float64(faker.Address().Longitude()),
+				ImageURL:    "ExampleURL",
```

Comment:
make the sample input into a URL format. Something like `https://exampleurl.com`. This is important because we can run validation to make sure URL is valid or not